conftest.py

```diff
@@ -0,0 +1,134 @@
+from django.contrib.auth import get_user_model
+from player.models import BallGame, Player
+from player_rating.models import PlayerRating, Rating
+from court.models import Court
+from game_event.models import GameEvent
+from decimal import Decimal
+from django.utils import timezone
+from message.models import Message
+from game_event_player.models import GameEventPlayer
+from court_ball_game.models import CourtBallGame
+import pytest
+
+TEST_BALL_GAME = 'Basketball'
+
+
+@pytest.fixture
+def user():
+    return get_user_model().objects.create_user(
+        username='testuser',
+        password='testpass')
+
+
+@pytest.fixture
+def user2():
+    return get_user_model().objects.create_user(
+        username='testuser2',
+        password='testpass')
+
+
+@pytest.fixture
+def user3():
+    return get_user_model().objects.create_user(
+        username='testuser3',
+        password='testpass')
+
+
+@pytest.fixture
+def player(user):
+    player = Player.objects.create(
+        user=user,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer)
+    return player
+
+
+@pytest.fixture
+def player2(user2):
+    player = Player.objects.create(
+        user=user2,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer)
+    return player
+
+
+@pytest.fixture
+def player3(user3):
+    player = Player.objects.create(
+        user=user3,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer)
+    return player
+
+
+@pytest.fixture
+def players(player, player2, player3):
+    return tuple([player, player2, player3])
+
+
+@pytest.fixture
+def player_rating(player):
+    player_rating_fixture = PlayerRating.objects.create(
+        ball_game=BallGame.Soccer,
+        user_id=player,
+        rating=Rating.Five)
+    return player_rating_fixture
+
+
+@pytest.fixture
+def court():
+    court = Court.objects.create(
+        x=Decimal('11'),
+        y=Decimal('22'),
+        city='TEST_CITY',
+        neighborhood='TEST_NEIGHBORHOOD',
+        max_players=30)
+    return court
+
+
+@pytest.fixture
+def game_event(court):
+    return GameEvent.objects.create(
+        id=2,
+        time=timezone.now(),
+        level_of_game=3,
+        min_number_of_players=2,
+        max_number_of_players=5,
+        court_id=court,
+        ball_game=TEST_BALL_GAME)
```

Comment:
If you don't need this constant don't forget to change the value of ball_game here

Reply:
thanks, will be fixed